File: backend/src/main/java/pl/edu/ur/blokur/BlokurApplication.java
package pl.edu.ur.blokur;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

/**
 * Punkt wejścia aplikacji Blokur.
 */
@SpringBootApplication
@EnableAsync
public class BlokurApplication {

    public static void main(String[] args) {
        SpringApplication.run(BlokurApplication.class, args);
    }
}

File: backend/src/main/kotlin/pl/edu/ur/blokur/BlokurApplication.kt
package pl.edu.ur.blokur

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(
    excludeName = [
        "org.springframework.boot.jdbc.autoconfigure.DataSourceAutoConfiguration",
        "org.springframework.boot.hibernate.autoconfigure.HibernateJpaAutoConfiguration"
    ]
)
class BlokurApplication

fun main(args: Array<String>) {
    runApplication<BlokurApplication>(*args)
}
